pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -26,104 +25,72 @@ import MultiSelect from "../common/multiSelect";
 import ElementCreationNew from "../common/elementCreationNew";
 
 interface HandlerFormProps {
-  id: string;
+  handlerId: string;
   endpointId: string;
 }
 
-export const HandlerForm: React.FC<HandlerFormProps> = ({ id, endpointId }) => {
-  const [context, setContext] = React.useState(null);
+export const HandlerForm: React.FC<HandlerFormProps> = ({ handlerId, endpointId }) => {
   const [handler, setHandler] = React.useState(null);
   const [showSpinner, setShowSpinner] = React.useState<boolean>(false);
   const [loadingOverlay, setLoadingOverlay] = React.useState<boolean>(false);
   const [entities, setEntities] = React.useState(null);
   const [tableNames, setTableNames] = React.useState<Array<any>>(null);
-  const title: string = id ? "Edit Handler" : "Create Handler";
+  const title: string = handlerId ? "Edit Handler" : "Create Handler";
   const API: APIService = React.useContext(APIContext);
   const [documentation, setDocumentation] = React.useState<string>(null);
   const [_, setAlert] = React.useContext(AlertContext);
   const [__, setHeader] = React.useContext(HeaderContext);
 
   React.useEffect(() => {
+    // handleSetDocumentation();
```

Comment:
When blocks of code aren't working, we might as well just delete them. 

Therefore, I suggest deleting this line of code and it's corresponding functions (starting on line `96`) and state (on line `40`).